extras/jsonrpc/daemon_types.go

```diff
@@ -471,6 +471,50 @@ type UTXOListResponse []struct {
 	Type          string `json:"type"`
 }
 
+type UTXOReleaseResponse *string
```

Comment:
Can just use a `string` here.

Reply:
Wasn't sure if that would handle this null response from the sdk: 

{
  "jsonrpc": "2.0",
  "result": null
}

Reply:
that might be problematic indeed. @eukreign why is the SDK returning null? Could this be improved?

Reply:
there is no return value from this function, i guess somewhere down the line `null` gets filled in as a default instead, why is it problematic? `go` doesn't support calling RPC with no result?